### DISCUSSION TITLE
Way to prevent nextJS from changing git branch name during project creation?

### DISCUSSION THREAD

**Discussion by arthur-zhuk** [vercel/next.js#45107] [Help]
Is there a way to prevent create-next-app from changing the git branch naming away from default without having to run `git branch -m`? Is there a flag or config that can be addressed?

**Comment by kfirfitousi** [comment]
Opened a PR that adds the option to specify the initial branch name when using create-next-app: https://github.com/vercel/next.js/pull/45154

**Reply by arthur-zhuk** [reply]
ty good sir
